packages/coachmark/test/coach-indicator.test.ts

```diff
@@ -61,27 +53,6 @@ describe('dev mode', () => {
         consoleWarnStub.restore();
     });
 
-    it('warns in Dev Mode when deprecated `static` attribute is used', async () => {
-        const el = await fixture<CoachIndicator>(html`
-            <sp-coach-indicator static="white"></sp-coach-indicator>
-        `);
-        await elementUpdated(el);
-        expect(consoleWarnStub.called).to.be.true;
-
-        const spyCall = consoleWarnStub.getCall(0);
-        expect(
-            (spyCall.args.at(0) as string).includes('deprecated'),
-            'confirm deprecated static warning'
-        ).to.be.true;
-        expect(spyCall.args.at(-1), 'confirm `data` shape').to.deep.equal({
-            data: {
-                localName: 'sp-coach-indicator',
-                type: 'api',
-                level: 'deprecation',
-            },
-        });
-    });
-
     it('warns in Dev Mode when deprecated `variant` attribute is used', async () => {
         const el = await fixture<CoachIndicator>(html`
             <sp-coach-indicator variant="white"></sp-coach-indicator>
```

Comment:
no variant?

Reply:
I've been rebasing this branch so you should start seeing some of these going away.